programs/clearing_house/src/state/oracle.rs

```diff
@@ -9,6 +9,34 @@ use std::cmp::max;
 use switchboard_v2::decimal::SwitchboardDecimal;
 use switchboard_v2::AggregatorAccountData;
 
+#[derive(Default, AnchorSerialize, AnchorDeserialize, Clone, Copy, Eq, PartialEq, Debug)]
+pub struct HistOracleData {
+    // use u64?
+    pub last_oracle_price: i128,
```

Comment:
if you wanna make these shorter, can maybe drop the `_oracle_` in the variable name since it's clear it's the oracle price as it's in the `HistOracleData`?